path-finding.asciidoc

```diff
@@ -12,3 +12,31 @@ Relevant questions to answer:
   * What information can be sent to intermediate and the final node aside from the critical routing data?
   * What are multi-hop locks? What addition privacy and security guarantees to they offer?
   * How can the flexible onion space be used to enabled packet switching in the network?
+
+
+=== What is "Source-Based" routing and why does the Lightning Network use it?
+
+Source-based routing is a method of path-finding where the sender (i.e. the source) plans the route from itself, through the intermediary nodes, and finally to the destination.
+Once a route has been selected, the sender sends the payment to the first intermediary node, who sends it to the second intermediary node and so on, until it reaches the destination.
+While a payment is in-flight along a route, the route cannot be changed by any of the intermediary nodes, even if a shorter path or a cheaper path (in terms of routing fees) exists.
```

Comment:
...along a path, the path cannot... 

though not sure if that is true. the current system technically allows for JIT routing which technically changes the path to some degree. at least it seems equivalent to link level AMP

Reply:
Amended. Hmm, in my understanding JIT routing is a different method to source-based routing. However, I'm certainly not well-versed in JIT routing so happy for you to suggest an alternate explanation here.

https://github.com/lnbook/lnbook/pull/309/commits/47491e4c166c9f9169a0d90ef3437c0857415c85